### PR TITLE
Add scrollsToTop as a BindingTarget on UIScrollView

### PR DESCRIPTION
**Pull request by Qata**

### CODE REVIEW

File: ReactiveCocoa/UIKit/UIScrollView.swift [hunk 1]
```diff
@@ -31,4 +31,9 @@ extension Reactive where Base: UIScrollView {
 	public var maximumZoomScale: BindingTarget<CGFloat> {
 		return makeBindingTarget { $0.maximumZoomScale = $1 }
 	}
+
+	/// Sets whether the scroll view scrolls to the top when the menu is tapped.
+	public var scrollsToTop: BindingTarget<Bool> {
```
Comment (mdiep) [inline ReactiveCocoa/UIKit/UIScrollView.swift]: It looks like this needs to be annotated to only be available on iOS (not tvOS).

https://travis-ci.org/ReactiveCocoa/ReactiveCocoa/jobs/244371741

**Comment by mdiep** [comment]
Thanks for the PR!

File: ReactiveCocoa/UIKit/UIScrollView.swift (hunk 1 above)
Comment (Qata) [inline ReactiveCocoa/UIKit/UIScrollView.swift]: Gotcha, I'll get that done.

**Comment by Qata** [comment]
@mdiep Changes done!

File: ReactiveCocoa/UIKit/UIScrollView.swift [hunk 2]
```diff
@@ -31,4 +31,10 @@ extension Reactive where Base: UIScrollView {
 	public var maximumZoomScale: BindingTarget<CGFloat> {
 		return makeBindingTarget { $0.maximumZoomScale = $1 }
 	}
+
+	/// Sets whether the scroll view scrolls to the top when the menu is tapped.
+	@available(tvOS, unavailable)
```
Comment (mdiep) [inline ReactiveCocoa/UIKit/UIScrollView.swift]: I think the proper way to annotate this is `@available(iOS 2.0, *)`.

Comment (Qata) [inline ReactiveCocoa/UIKit/UIScrollView.swift]: Ah my bad.

**Comment by mdiep** [comment]
I thought that would work, but it still fails CocoaPods listing. @ikesyo @andersio?

**Comment by Qata** [comment]
That last one should fix it.

**Comment by mdiep** [comment]
Thanks for the PR!

**Pull request merged**